programs/protocol-contracts-solana/src/lib.rs

```diff
@@ -414,6 +428,51 @@ pub struct UpdatePaused<'info> {
     pub signer: Signer<'info>,
 }
 
+#[derive(Accounts)]
+pub struct AddToWhitelist<'info> {
```

Comment:
We should use consistent terminology with Solidity contracts:

- whitelist
- unwhitelist

Reply:
changed to whitelist/unwhitelist across the board
e905e678536ab80badbd9046137df10c02c15fcc